### DISCUSSION TITLE
need m3u links for youtube

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#973] [Q&A]
hello 
im looking for m3u links for the algerian radio livestream on youtube . 
here is the links of channels 
chaine 1: https://www.youtube.com/watch?v=Ykl6f7rNM0k
chaine 2 : https://www.youtube.com/@radiochainedeuxALG
chaine 3: https://www.youtube.com/@radioalgerienneCH3
chaine inter : https://www.youtube.com/@radioalginter

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
